### DISCUSSION TITLE
Cannot upload jpgs to issues

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#2815] [Mobile]
I have to use the website to add a JPGs. They won't upload with the app.
![image](https://user-images.githubusercontent.com/1230959/100495263-a2b2c100-3184-11eb-8266-c3c5541cbd03.png)

**Comment by candyho** [comment]
Thanks for flagging! This should be fixed in the latest version - let us know if this still happens after an app update!

**Reply by jidanni** [reply]
Bad news: bug still present in version 1.3.14.
Please tell me which version number is the one that contains the fix. Thank you.

**Reply by candyho** [reply]
Hmmm that's weird. Were you trying to create a new issue by sharing the image with the app or did you pick the image inside of the dialog using the image icon?

**Comment by jidanni** [comment]
I never share images with the app.
I always hit the "+" to create an issue.
Then press the add image icon to add an image.

**Reply by candyho** [reply]
Thanks! Unfortunately we couldn't seem to reproduce this bug and we suspect this is an Android 8-specific issue (from your other reply you shared you're on an Android 8 device)... are you experiencing similar image upload issue with other apps on the same device?

**Reply by jidanni** [reply]
Today I could consistently upload this image:
![Screenshot_20201204-140218](https://user-images.githubusercontent.com/1230959/101129421-7c859900-363c-11eb-9a1f-eb5ec5d66254.jpg)


 but for the following one, I consistently got the failure message:
![Screenshot_20201204-141735](https://user-images.githubusercontent.com/1230959/101129249-3b8d8480-363c-11eb-8431-b21e54357ee5.jpg)

Okay, one is 100 kilobytes the other is 200 kilobytes. So maybe we're talking about some kind of network problem.

**Reply by jidanni** [reply]
(But how could it be that I could use the browser to do it successfully but not the app? If it was a network speed timeout problem both would fail.)

**Comment by jidanni** [comment]
Okay, standing in the same spot with my same cell phone I was able to upload the image as you can see on https://github.com/jidanni/mymaps/issues/2 .
However, as I also recorded, there above it, two previous attempts to upload much larger images.

**Comment by jidanni** [comment]
Well in a pinch I can upload PNGs instead. Bigger, but at least no bug.

**Comment by JingyuZhang-git** [comment]
PNG might be used.
